src/routes/sessions.ts

```diff
@@ -0,0 +1,56 @@
+import { CodeBox } from "@ory/elements-markup"
+import {
+  defaultConfig,
+  requireAuth,
+  RouteCreator,
+  RouteRegistrator,
+} from "../pkg"
+import { navigationMenu } from "../pkg/ui"
+
+export const createSessionsRoute: RouteCreator =
+  (createHelpers) => async (req, res) => {
+    const { sdk } = createHelpers(req)
+    const session = req.session
+
+    // Create a logout URL
+    const logoutUrl =
+      (
+        await sdk
+          .createSelfServiceLogoutFlowUrlForBrowsers(req.header("cookie"))
+          .catch(() => ({ data: { logout_url: "" } }))
+      ).data.logout_url || ""
+
+    res.render("session", {
+      layout: "welcome",
+      traits: {
+        ...session?.identity.traits,
+        ...{ created_at: session?.identity.created_at || "" },
+        ...session?.authentication_methods?.reduce<any>(
```

Comment:
This reducer is really hard to read. If I read it correctly, this would produce the following entries in `traits`:

```js
{
  //... other traits
  "authentication_method: 4 Jun 2017 00:00:00 PDT": "password",
  //... other traits
}
```

Do we need that key to use it as a look-up? That seems _really_ hacky and awkward to me.

Reply:
this is indeed hacky and was rushed a bit here to get the UI working as I wanted it to.

The keys are just used the UI to display the traits in a "form-like" structure.

Reply:
Here is an example

![image](https://user-images.githubusercontent.com/18033717/192269724-91d3f3dd-f131-4f2d-9b58-9ac53eadea3b.png)